test/e2e/network/netpol/model.go

```diff
@@ -90,7 +90,7 @@ func NewModel(namespaces []string, podNames []string, ports []int32, protocols [
 
 // GetProbeTimeoutSeconds returns a timeout for how long the probe should work before failing a check, and takes windows heuristics into account, where requests can take longer sometimes.
 func (m *Model) GetProbeTimeoutSeconds() int {
-	timeoutSeconds := 1
+	timeoutSeconds := 3
```

Comment:
Echoing https://github.com/kubernetes/kubernetes/pull/104087#issuecomment-891728927 -- what prompted this?

And also, if we're going to use the same value across OSs, we can drop the windows conditional, and drop the `GetProbeTimeoutSeconds` method altogether in favor of a named constant.

Reply:
just testing the CI, this PR wont be merged